src/EnergyPlus/CurveManager.cc

```diff
@@ -3385,20 +3412,21 @@ namespace CurveManager {
 				gio::write( OutputFileInits, Format_130 );
 				gio::write( OutputFileInits, Format_150 ) << CurrentModuleObject << Alphas( 1 );
 			}
-			NumIVars = Numbers( 15 );
-			if ( NumIVars > 5 || NumIVars < 1 ) {
+			NumIVars = Numbers( 17 );
```

Comment:
Grab the number of independent variables from two N lower, and adjust the `BaseOffset` a few lines below up by 2 also.